lib/cosmos/gui/utilities/script_module_gui.rb

```diff
@@ -71,6 +71,32 @@ module Script
 
     @@qt_boolean = Qt::Boolean.new
 
+    def _get_main_thread_gui
+      result = nil
+      Qt.execute_in_main_thread(true, 0.05) do
+        window = nil
+        window = get_cmd_tlm_gui_window() if get_cmd_tlm_gui_window()
```

Comment:
Approved.  Please remove the window = nil line and just leave window = get_cmd_tlm_gui_window()